compiler/circle-partitioner/src/PartitionJson.cpp

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PartitionJson.h"
+#include "PartitionPath.h"
+
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace
+{
+
+const char *_CLF = "\n"; // Control Line Feed
+const char *_DQU = "\""; // Double QUotation
+const char *_COM = ",";  // COMma
+const char *_SPC = " ";  // SPaCe
```

Comment:
I think these are not needed because...

- Actually they are all `char`, not `char *`(string).
- Using real character seems more clear and easier to read than using alias.
- They are only used for printing.
- No effect to reduce code duplication

Reply:
> Actually they are all char, not char *(string).

You got a point. But this is my choice.

> Using real character seems more clear and easier to read than using alias.

Adding this was from my oposite code. it made me reading difficult.

> No effect to reduce code duplication

There are some but this was not the main point. Reading code was difficult.

Reply:
using `char` seems to work. started with `char *` as I thought to replace a string not just one character but tuned out like this.
I'll be working on this and may revise this.